service/controller/azure_config.go

```diff
@@ -0,0 +1,642 @@
+package controller
```

Comment:
This isn't new. This is the controller that we already had to watch `AzureConfig` CRs, but renamed from `cluster.go` to `azure_config.go`.